client/java/src/main/java/com/linecorp/centraldogma/client/WatchOptions.java

```diff
@@ -0,0 +1,46 @@
+package com.linecorp.centraldogma.client;
```

Comment:
missing copyright, but we don't need this class anymore so no big deal. 😅 
You can check if there are any style issues at your local using Gradle:
```bash
./gradlew clean build --parallel --no-daemon --max-workers=4
```
This is what I usually do. 😉